Source/WebKit/GPUProcess/graphics/WebGPU/RemoteTexture.cpp

```diff
@@ -74,6 +74,7 @@ void RemoteTexture::createView(const std::optional<WebGPU::TextureViewDescriptor
 void RemoteTexture::destroy()
 {
     m_backing->destroy();
+    m_objectHeap.removeObject(m_identifier);
```

Comment:
Ditto. Destroy() and Destruct() should be different messages, with different implementations.